packages/astro/src/core/ssr/index.ts

```diff
@@ -29,7 +29,7 @@ import { injectTags } from './html.js';
 import { generatePaginateFunction } from './paginate.js';
 import { getParams, validateGetStaticPathsModule, validateGetStaticPathsResult } from './routing.js';
 
-const svelteAndVueStylesRE = /\?[^&]+&type=style&lang/;
+const svelteStylesRE = /svelte\?svelte&type=style/;
```

Comment:
Related change: this was missing Vue styles, which is fine because Vue works with either a `script` or `link` tag (incredible). This change keeps the exact same behavior, just makes its purpose more clear.